config/locales/en.yml

```diff
@@ -46,6 +46,10 @@ en:
           attributes:
             end_date:
               greater_than_or_equal_to: "must be on or after start date"
+        assignment/rotation_generator:
+          attributes:
+            end_date:
+              greater_than_or_equal_to: "must be on or after start date"
```

Comment:
We can make this a little more general/shareable - I've left a lot already so I'm not gonna get into how we should do this but just leaving a note for later.